src/index.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+
 import './css/index.css';
+import { configureStore } from '@reduxjs/toolkit';
+import reducer from './redux/configureStore';
 import App from './App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
+
+const store = configureStore({
+  reducer,
+});
```

Comment:
- [ ] Your app is looking great, well done! Reducers and Actions are set up properly, however, I was not able to add or remove a book:

![Captura de pantalla 2022-06-06 a la(s) 10 22 23 p  m](https://user-images.githubusercontent.com/83717285/172291315-b94a7bcc-eab5-467c-b3d5-1ee336dbe6d9.png)

It would be great to check those functionalities in order to be able to add/remove books.

If it's working on your machine, can you please upload a screenshot to this PR to see if it's a general problem, or is it just in my machine?

Tip: I was looking at your store and I couldn't find where is the `createStore` imported from `redux` to create your store, I see you are using `configureStore`, just check if it has something to do with that.

Reply:
Thank you so much for the feedback @smunozmo, I thought that adding and removing from the UI wasn't a requirement for this milestone but I guess I was mistaken, I added the functionality, and it should be working now, I'll ask for another review.